docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md

```diff
@@ -110,77 +114,96 @@ In this section, we will:
+- A basic CanJS application is a live-bound template (or view) rendered with a component's ViewModel.
 
 - A [can-stache] template is used to render data into a document fragment:
 
   ```js
-  const template = can.stache("<h1>{{message}}</h1>");
-  const fragment = template({message: "Hello World"});
-  fragment //-> <h1>Hello World</h1>
+  import { Component } from "can";
+
+  Component.extend({
+    tag: "my-app",
+    view: `<h1>{{ message }}</h1>`,
+    ViewModel: {
+      message: {
+        default: () => "Hello World"
+      }
+    }
+  });
   ```
 
-- Load a template from a `<script>` tag with [can-stache.from can.stache.from] like:
+- Mount a [can-component] by its [can-component.prototype.tag custom tag] like:
 
-  ```js
-  const template = can.stache.from(SCRIPT_ID);
+  ```html
+  <my-app></my-app>
   ```  
 
-- Use [can-stache.helpers.if {{#if(value)}}] to do `if/else` branching in `can-stache`.
+- Use [can-stache.helpers.if {{# if(value) }}] to do `if/else` branching in `can-stache`.
 - `Promise`s are observable in `can-stache`.  Given a promise `somePromise`, you can:
   - Check if the promise is loading like: `{{#if(somePromise.isPending)}}`.
   - Loop through the resolved value of the promise like: `{{#for(item of somePromise.value)}}`.
 
 
-- [can-define/map/map can.DefineMap] can be used to define the behavior of observable objects like:
+- [can-define/map/map DefineMap] can be used to define the behavior of observable objects like:
 
   ```js
-  const Type = can.DefineMap.extend({
-	  message: "string"
+  import { DefineMap } from "can";
+
+  const Type = DefineMap.extend({
+    message: "string"
   });
   ```
 
-- Instances of these [can-define/map/map can.DefineMap] types are often used
-  as a ViewModel that controls the behavior of a [can-stache] template (or
-  [can-component]).
+- Instances of these [can-define/map/map DefineMap] types are often used
+  as a ViewModel that controls the behavior of a [can-component].
 
   ```js
-  const PlaylistVM = can.DefineMap.extend({
-	  message: "string"
-  });
+  import { Component } from "can";
 
-  const messageVM = new PlaylistVM();
-  const fragment = template(messageVM)
+  Component.extend({
+    tag: "playlist-editor",
+    view: `...`,
+    ViewModel: {
+       message: "string"
+    }
+  });
   ```
 
-- `can.DefineMap` can specify a default value and a type:
+- `DefineMap` can specify a default value and a type:
   ```js
-  const PlaylistVM = can.DefineMap.extend({
-    count: {default: 33}
+  import { Component } from "can";
+
+  Component.extend({
+    tag: "playlist-editor",
+    view: `...`,
+    ViewModel: {
+       message: "string",
+       default: () => "Hello World"
+    }
   });
-  new PlaylistVM().count //-> 33
   ```
 
 
 ### The solution
 
-Update the __HTML__ tab by deleting everything inside the `<body>` and replacing
+Update the __HTML__ tab by deleting everything and replacing
```

Comment:
Is there anything they have to delete?

Reply:
Yes, the guide starts with HTML prototype

Reply:
Ah, I’ve been removing that from all the starters because they’re not necessary.

Reply:
Updated